### PR TITLE
Add Automatically Create Default `config.json`

### PR DESCRIPTION
**Pull request by outercloudstudio**

### CODE REVIEW

**Comment by FrederoxDev** [comment]
Majority looks good, I've left one comment inside

**Review by FrederoxDev — CHANGES_REQUESTED** [review]

File: AmethystLauncher/src/loader/Loader.cpp [hunk 1]
```diff
@@ -95,9 +91,18 @@ void ModLoader::getMinecraftWindowHandle()
...
-void ModLoader::InjectRuntime()
-{
-	// Get the path to the amethyst runtime dll
+void ModLoader::InjectRuntime() {
+    std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
+
+    if (mConfig.injectedMod.length() == 0) {
+        std::string message = fmt::format(
+            "No injected mod specified. Make sure to update your config.json "
+            "with the latest Amethyst Runtime.");
```
Comment (FrederoxDev) [inline AmethystLauncher/src/loader/Loader.cpp:100]: Does having two strings side by side work?

Comment (outercloudstudio) [reply AmethystLauncher/src/loader/Loader.cpp:100]: Yes, although let me remove that.

Comment (outercloudstudio) [reply AmethystLauncher/src/loader/Loader.cpp]: Okay removed

Comment (FrederoxDev) [reply AmethystLauncher/src/loader/Loader.cpp]: Huh lol, weird

**Review by FrederoxDev — APPROVED** [review]
LGTM!

**Pull request merged**